nilearn/__init__.py

```diff
@@ -8,67 +8,18 @@
 
 See http://nilearn.github.com for complete documentation.
 """
+import gzip
 
-from version import __version__
+from .version import check_module_dependencies
```

Comment:
Within a package, I always try to do relative imports to avoid any possible import conflicts.  Is this OK here?